# Omics analysis for beginners
## Correspondent:
  * Principal investigator: Gladys Alexandre - galexan2@utk.edu
  * Lab manager: Elena Ganusov - eganusov@utk.edu
  * Data analyst: Lam Vo - lvo5@vols.utk.edu (Personal: lamvo1998@gmail.com)

## Purpose:
Multi-omics analyses are becoming the gold standards for answering systems-level biological questions in many model organism. However, acquiring the computational skillset required for omics analyses remains a challenge for many scientists who are not traditionally trained in bioinformatics. Here, our lab offers step-by-step guidelines and example codes for analyzing big transcriptomics, proteomics, metabolomics, and methylomics datasets. All example codes provided are made from **Python versions 3.**

This guidelines will provide the user with knowledge on: 
* 1) Organizing, cleaning, and pre-processing big biological dataset for downstream analyses.
* 2) Performing statistical analyses (Student's t-test, Bonferroni correction, ...) to determine differentially expressed elements.
* 3) Creating graphs for exploratory and inferences purposes (principal component analyses, heatmaps of differentially expressed elements, dendrograms, ...)

This project is based on our proteomics analysis to infer the relationships of nitrogen and carbon metabolism to chemotaxis in *Azosprillum brasilense*. If interested, please refer to:
> (cite here)

<p align="center"> 
<img src="http://alexandrelab.utk.edu/images/Slide.jpg">
</p>

## Prerequisite:
This project assumes that the user is familiar with the Python's syntax. However, no prior coding experience is required to follow this guideline. Python 3 is required, and Jupyter Notebook is recommended. The instructions for installations are listed with the following link:
* https://realpython.com/installing-python/
* https://jupyter.readthedocs.io/en/latest/install.html 

